quiver/constants.py

```diff
@@ -8,8 +8,8 @@
     "QUIVER_MAIN",
     "RESULTS"
 ]
-
-METS = '{http://www.loc.gov/METS/}'
-OCRD = '{https://ocr-d.de}'
+from ocrd_models.constants import NAMESPACES
+METS = "{" + f"{NAMESPACES['mets']}" + "}"
```

Comment:
If you use `+`, the `f'{...}` in the middle seems unnecessary.

Reply:
Addressed in [edd81d6](https://github.com/OCR-D/quiver-back-end/pull/43/commits/edd81d68c561147770c715019b539b9d9aa24ad1)